uberpoet/dotreader.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, modules_filter=None):
             # We are trying to simulate a non-test app build and ignore non-code targets such as asset
             # catalogs and schemes. Each module currently gets a static amount of code created for it,
             # so non-code modules will add code to the total when they shouldn't if we didn't filter them out.
-            modules_filter = ['test', 'scheme', 'assetcatalog', 'resources', 'fixture', 'needle']
+            modules_filter = ['test', 'scheme', 'assetcatalog', 'resources', 'fixture', 'needle', 'assets']
```

Comment:
we use this naming for various modules in our codebase, I guess its not so bad to include this here for everything?

Reply:
sgtm